aes/src/soft/ctr.rs

```diff
@@ -1,5 +1,7 @@
 //! AES in counter mode (a.k.a. AES-CTR)
 
+#![cfg_attr(feature = "autodetect", allow(dead_code))]
+
```

Comment:
Is it a temporary thing until you add detection to CTR types?

Reply:
Yeah, CTR probably needs similar gating/wrappers. Should've left a TODO.

Reply:
CTR now uses `aes`+`ssse3` runtime detection and this line of code has been removed.